contracts/interfaces/modules/royalty/policies/IRoyaltyPolicy.sol

```diff
@@ -1,8 +1,10 @@
 // SPDX-License-Identifier: BUSL-1.1
 pragma solidity 0.8.26;
 
+import { IExternalRoyaltyPolicy } from "./IExternalRoyaltyPolicy.sol";
+
 /// @title RoyaltyPolicy interface
-interface IRoyaltyPolicy {
+interface IRoyaltyPolicy is IExternalRoyaltyPolicy {
```

Comment:
Can merge IRoyaltyPolicyLAP and IRoyaltyPolicyLRP interfaces into IRoyaltyPolicy, as interface IRoyaltyPolicyLAP and IRoyaltyPolicyLRP share the same functions.

Reply:
I know that seen as it is now - it's a valid point.

The reason they were not merged into a single IRoyaltyPolicy is because there are functions that only apply to "Full-tree awareness" calculation policies such as `transferToVault`. LAP and LRP interfaces look very similar because they both are "Full-tree awareness". 

But if a new 3rd policy came along it might not be so IRoyaltyPolicy contains only the functions that any royalty policy really needs to have.

Reply:
I see, 
In that case, could we merge IRoyaltyPolicyLAP and IRoyaltyPolicyLRP into one interface say name like "FullTreeAwarePolicy" which inherent from IRoyaltyPolicyLAP, so that we can reduce the duplicate?

Reply:
addressed in https://github.com/storyprotocol/protocol-core-v1/pull/215/commits/ec81ec2d3c039d54063ec15698ca799bb65f9edc